src/platform/common/platform/fileSystem.ts

```diff
@@ -101,4 +109,64 @@ export class FileSystem implements IFileSystem {
         const data = typeof text === 'string' ? Buffer.from(text) : text;
         return this.vscfs.writeFile(uri, data);
     }
+
+    async createTemporaryFile(options: { fileExtension?: string; prefix?: string }): Promise<TemporaryFileUri> {
+        // In non node situations, temporary files are created in the globalStorageUri location (extension specific)
```

Comment:
This seems to be in node context as well since it's in a common file. Does globalStorageUri mean different things in those contexts, or will this only be called in web?

Reply:
No it works for both. It's intended to work for both (using it in the tests now). globalStorageUri is just a location on disk for desktop, and it's an in memory location for web.

I should change the comment though.